tests/validators/test_typed_dict.py

```diff
@@ -808,6 +808,23 @@ def test_from_attributes(input_value, expected):
         assert output == expected
 
 
+def test_from_attributes_kwargs():
+    v = SchemaValidator(
+        {
+            'type': 'typed-dict',
+            'fields': {
+                'a': {'schema': {'type': 'int'}},
+                'b': {'schema': {'type': 'int'}},
+                'c': {'schema': {'type': 'str'}},
+            },
+            'from_attributes': True,
+        }
+    )
+    test_input = dict(a=1, b=2, c='3', d=True)
```

Comment:
your input data passed is still a dict, not a tuple.

I think you need to pass something like 

```py

    class Foobar:
        def __init__(self):
            self.a = 1
            self.b = 2
            self.c = 3

test_input = (Foobar(), dict(c=3)
```